ts/components/services/ServicesNational.tsx

```diff
@@ -1,6 +1,7 @@
+// TODO: delete this component (bieng substituted with the generic ServicesSectionsList)
```

Comment:
each TODO should be referring to a story

Reply:
I removed the file within this pr, they are no more used (4a5010d53af5dcbf687bc50cc6ec1cff542f3554)